### DISCUSSION TITLE
How to contact github management

### DISCUSSION THREAD

**Discussion by X4CARD** [community/community#119652] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How i connect to github mgmt?

**Comment by ebndev** [comment]
Hi @X4CARD,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions is by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
